app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from "react";
+import { fetchGreeting } from "./redux/greeting";
```

Comment:
- [OPTIONAL] I think this is the wrong path for importing, the good path is `import { fetchGreeting } from "../redux/greeting";`. Please kindly fix that 🙃 😁